test/virtual-dom/index.ts

```diff
@@ -0,0 +1 @@
+import './core';
```

Comment:
No new line EOF. This is needed because of Git diffs. You should set this as a standard so you need not to think about it.